libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -582,12 +583,31 @@ public synchronized String getEncryptionKeyForPasscode(String actualPass) {
     }
 
     /**
+     * Returns the branded login path.
+     *
+     * @return Branded login path, or empty string if not configured.
+     */
+    public String getBrandedLoginPath() {
```

Comment:
Should we just call it brand?
The name is a bit misleading. This method doesn't return the full path of the login URL.